plugins/inputs/intel_powerstat/sample.conf

```
@@ -22,7 +22,29 @@
   ## by the plugin.
   ## Supported options:
   ##   "cpu_frequency", "cpu_c0_state_residency", "cpu_c1_state_residency",
-  ##   "cpu_c6_state_residency", "cpu_busy_cycles", "cpu_temperature",
-  ##   "cpu_busy_frequency"
-  ## ATTENTION: cpu_busy_cycles is DEPRECATED - use cpu_c0_state_residency
+  ##   "cpu_c3_state_residency", "cpu_c6_state_residency", "cpu_c7_state_residency",
+  ##   "cpu_temperature", "cpu_busy_frequency", "cpu_c0_substate_c01",
+  ##   "cpu_c0_substate_c02", "cpu_c0_substate_c0_wait"
   # cpu_metrics = []
+
+  ## Optionally the user can choose for which CPUs metrics configured in cpu_metrics array should be gathered.
+  ## Can't be combined with excluded_cpus.
+  ## Empty or missing array means CPU metrics are gathered for all CPUs.
+  ## e.g. ["0-3", "4,5,6"] or ["1-3,4"]
+  # included_cpus = []
+
+  ## Optionally the user can choose which CPUs should be excluded from gathering metrics configured in cpu_metrics array.
+  ## Can't be combined with included_cpus.
+  ## Empty or missing array means CPU metrics are gathered for all CPUs.
+  ## e.g. ["0-3", "4,5,6"] or ["1-3,4"]
+  # excluded_cpus = []
+
+  ## Filesystem location of JSON file that contains PMU event definitions.
+  ## Mandatory only for perf-related metrics (cpu_c0_substate_c01, cpu_c0_substate_c02, cpu_c0_substate_c0_wait).
+  # event_definitions = ""
+
+  ## The user can set the timeout duration for MSR reading.
+  ## Enabling this timeout can be useful in situations where, on heavily loaded systems,
+  ## the code waits too long for a kernel response to MSR read requests.
+  ## 0 disables the timeout (default).
+  # msr_read_timeout = 0
```

Since it is a duration suggest to specify it in seconds? (Or would ms be more logical, that's fine as well)
```suggestion
  # msr_read_timeout = 0s
```